UnitTests/GitCommands.Tests/Git/Commands/GitCommandHelpersTest.cs

```diff
@@ -499,15 +499,17 @@ public void RebaseCmd()
                 () => GitCommandHelpers.RebaseCmd("branch", false, false, false, false, false, false, from: "from", onto: null));
         }
 
-        [TestCase(CleanMode.OnlyNonIgnored, true, false, null, "clean --dry-run")]
-        [TestCase(CleanMode.OnlyNonIgnored, false, false, null, "clean -f")]
-        [TestCase(CleanMode.OnlyNonIgnored, false, true, null, "clean -d -f")]
-        [TestCase(CleanMode.OnlyNonIgnored, false, false, "paths", "clean -f paths")]
-        [TestCase(CleanMode.OnlyIgnored, false, false, null, "clean -X -f")]
-        [TestCase(CleanMode.All, false, false, null, "clean -x -f")]
-        public void CleanCmd(CleanMode mode, bool dryRun, bool directories, string paths, string expected)
-        {
-            Assert.AreEqual(expected, GitCommandHelpers.CleanCmd(mode, dryRun, directories, paths).Arguments);
+        [TestCase(CleanMode.OnlyNonIgnored, true, false, null, null, "clean --dry-run")]
+        [TestCase(CleanMode.OnlyNonIgnored, false, false, null, null, "clean -f")]
+        [TestCase(CleanMode.OnlyNonIgnored, false, true, null, null, "clean -d -f")]
+        [TestCase(CleanMode.OnlyNonIgnored, false, false, "paths", null, "clean -f paths")]
+        [TestCase(CleanMode.OnlyNonIgnored, false, false, "paths", "-e /excludes", "clean -f paths -e /excludes")]
+        [TestCase(CleanMode.OnlyNonIgnored, false, false, null, "-e /excludes", "clean -f -e /excludes")]
+        [TestCase(CleanMode.OnlyIgnored, false, false, null, null, "clean -X -f")]
+        [TestCase(CleanMode.All, false, false, null, null, "clean -x -f")]
+        public void CleanCmd(CleanMode mode, bool dryRun, bool directories, string paths, string excludes, string expected)
```

Comment:
Can you please add test cases for multiple includes and excludes too?

Reply:
I'm not sure of the best way to do this. The parameters passed in here for testing, ("paths") and ("excludes"), are functionally equivalent to any amount of paths and excludes as represented in the form of a single string. I could modify the unit tests to use a string parameter like:

(`"\\"path1\\" \\"path2\\""`) 

 to make the formatting explicit if that would be beneficial. I could also just do (`"path1 path2"`).

For now I've gone ahead and removed the "-e /" formatting from the excludes test parameters to bring it in line with the way the include paths test parameter is set up.

Reply:
We can always add new tests for specific scenarios :)